lib/galaxy/tool_util/parser/cwl.py

```diff
@@ -134,7 +138,7 @@ def _parse_output(self, tool, output_instance):
         output.metadata_source = ""
         output.parent = None
         output.label = None
-        output.count = None
+        output.count = 1
```

Comment:
`count` initialized as `output_dict.get("count", 1)` in other places, probably good here too